packages/vega-label/src/util/placeAreaLabel.js

```diff
@@ -1,138 +0,0 @@
-import {textMetrics} from 'vega-scenegraph';
```

Comment:
Did I delete this file when I rebased or did you remove it @chanwutk?

Reply:
I will check.

Reply:
I deleted it. The file is not used anywhere.